crates/sui-rosetta/tests/end_to_end_tests.rs

```diff
@@ -441,7 +441,8 @@ async fn test_pay_sui_multiple_times() {
 
     let (rosetta_client, _handle) = start_rosetta_test_server(client.clone()).await;
 
-    for _ in 1..100 {
+    for i in 1..20 {
+        println!("Iteration: {}", i);
```

Comment:
why was the iterations reduced? Also do you want to use `tracing::info!` here instead?

Reply:
This test takes >3min unless min round delay is increased to 500ms (Narwhal level). I think it may be related to per checkpoint processing cost. `println!()` is common elsewhere in the rosetta codebase.